packages/metrics-dotnet/MutationMetrics/Metrics.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Mutation.Metrics.Core.Models;
+using System.Text.Json;
+
+namespace Mutation.Metrics.Core
+{
+    public static class Metrics
+    {
+        public static MetricResult GenerateScore(string fullPath)
+        {
+            try
+            {
+                double killed = 0;
+                double timeOut = 0;
+                double survived = 0;
+                double noCoverage = 0;
+                double runtimeErrors = 0;
+                double compileErrors = 0;
+                double ignored = 0;
+                double pending = 0;
+
+                using StreamReader reader = new(fullPath);
+                var reporterFile = reader.ReadToEnd();
+
+                JsonReport report = JsonSerializer.Deserialize<JsonReport>(reporterFile, new JsonSerializerOptions { PropertyNameCaseInsensitive = true });
```

Comment:
I'm not sure this belongs here, CalculateMetrics should probably receive a concrete class. We could perhaps provide a helper class for deserializing depending on the conversation on if this would be within scope for this package.